packages/signal-polyfill/src/wrapper.spec.ts

```diff
@@ -188,7 +204,7 @@ describe("Watcher", () => {
     // Adding any other effect after an unwatch should work as expected
     const destructor2 = effect(() => {
       output = stateSignal.get();
-      return () => {};
+      return () => { };
```

Comment:
apologies -- my prettier plugin considers the existence of package.json as "when to enable prettier" :(

https://github.com/stevearc/conform.nvim/issues/268

I've opened this as well: https://github.com/proposal-signals/proposal-signals/pull/163